src/datascience-ui/interactive-common/redux/store.ts

```diff
@@ -159,6 +156,7 @@ function createMiddleWare(
     const testMiddleware =
         forceOnTestMiddleware || testMode || isUITest ? createTestMiddleware(transformLoad) : undefined;
 
+    /* Fixup this code if you need to debug redux
```

Comment:
I think some of our tests rely on this middleware logging still.

Reply:
I guess we'll see when the test runs come in, but I think some of the variable tests wait for variable complete logging from the middleware.

Reply:
From what I know, the process environment variables were never set to anything, so unless I misread the logic here, the testMiddleware isn't even possible to enable.

Reply:
Hmm, I think I was thinking of the forceTestMiddleware part of webview.ts. Does that still work? Pretty sure that was checking ENV.

Reply:
Ahh, that's still on the main node side. Nevermind.